android/src/main/java/io/github/droidkaigi/feeder/ArborInitializer.kt

```diff
@@ -3,11 +3,11 @@ package io.github.droidkaigi.feeder
 import android.content.Context
 import androidx.startup.Initializer
 import com.toxicbakery.logging.Arbor
-import com.toxicbakery.logging.LogCatSeedling
+import com.toxicbakery.logging.Seedling
 
 class ArborInitializer : Initializer<Unit> {
     override fun create(context: Context) {
-        Arbor.sow(LogCatSeedling())
+        Arbor.sow(Seedling())
```

Comment:
Thank you! Perhaps only the android module should have depended on android abor.
I will try to make another issue. 👍